<launch>
  <!-- raspicam_node package should be downloaded and be launched in the raspberry pi before launch this file -->
  <!-- <arg name="model" default="$(env TURTLEBOT3_MODEL)" doc="model type [burger, waffle, waffle_pi]"/>

  <include file="$(find turtlebot3_bringup)/launch/includes/description.launch.xml">
    <arg name="model" value="$(arg model)" />
  </include>

  <node pkg="joint_state_publisher" type="joint_state_publisher" name="joint_state_publisher">
    <param name="use_gui" value="true"/>
    <param name="rate" value="50"/>
  </node>

  <node pkg="robot_state_publisher" type="robot_state_publisher" name="robot_state_publisher">
    <param name="publish_frequency" type="double" value="50.0" />
  </node> -->
  <!-- <arg name="model" default="$(find hw_t)/urdf/wheelcheck.urdf.xacro"/>
  <node pkg="rviz" type="rviz" name="rviz" 
    args="-d $(find turtlebot3_automatic_parking_vision)/rviz/turtlebot3_automatic_parking_vision.rviz"/>
  <node pkg="tf" type="static_transform_publisher" name="camera_link" 
    args="0 0 0 0 0 0 /map camera_link 10" /> -->

	<node pkg="ar_track_alvar" type="individualMarkersNoKinect" name="ar_track_alvar" respawn="false" output="screen">
		<param name="marker_size"           type="double" value="5.2" />
		<param name="max_new_marker_error"  type="double" value="0.08" />
		<param name="max_track_error"       type="double" value="0.2" />
		<param name="output_frame"          type="string" value="camera_link" />

		<remap from="camera_image"  to="/camera/color/image_raw" />
		<remap from="camera_info"   to="/camera/color/camera_info" />
	</node>

  <node pkg="turtlebot3_automatic_parking_vision" type="automatic_parking_vision" name="automatic_parking_vision" />
</launch>
